testing/run-scripts/run_cloud.sh

```diff
@@ -64,7 +65,17 @@ then
     fi
     BANNER=`docker exec uproxy-sshd cat /banner`
   else
-    BANNER=`hostname`
+    # DigitalOcean's metadata API can tell us the region in which
+    # the machine is located:
+    #   https://developers.digitalocean.com/documentation/metadata/#metadata-api-endpoints
+    BANNER=`curl -s -m 2 http://169.254.169.254/metadata/v1/region`
+    if [ ! -z "$BANNER" ]
+    then
+      BANNER=`echo "$BANNER"|sed 's/ams./Netherlands/;s/sgp./Singapore/;s/fra./Germany/;s/tor./Canada/;s/nyc./New York, USA/;s/sfo./San Francisco, USA/;s/lon./UK/'`
+      BANNER="$BANNER (DigitalOcean)"
```

Comment:
Good ideas but, yes, I'm reluctant to make it any much more cleverer than it already is. I think we're still at the stage where anyone running >1 server will be able to configure their own descriptions and I wanna keep the instructions simple for everyone else.

I can imagine a followup post of tips and tricks, such as manually changing your server's description.